experiments/AtmosLES/bomex.jl

```diff
@@ -562,7 +416,18 @@ function config_bomex(FT, N, resolution, xmax, ymax, zmax)
         turbulence = SmagorinskyLilly{FT}(C_smag),
         moisture = EquilMoist{FT}(; maxiter = 5, tolerance = FT(0.1)),
         source = source,
-        boundarycondition = bc,
+        boundarycondition = (
+            AtmosBC(
+                momentum = Impenetrable(DragLaw(
+                    (state, aux, t, normPu_int) -> (u_star / normPu_int)^2,
```

Comment:
Perhaps a comment / clarification that the notation `P` represents the horizontal terms; (since the vertical boundary velocity is zero anyway for the impenetrable condition)